src/screens/balanceWithdrawal/WithdrawAddress.tsx

```diff
@@ -30,8 +29,9 @@ type HandleConfirm = {
 
 export default function WithdrawAddress() {
   const [address, setAddress] = useState("");
-  const [transactionRespose, setTransaciontRespose] = useState("");
-  const [withdrawAsset, setWithdrawAsset] = useAtom(withdrawAtom);
+  const [transactionRespose, setTransaciontRespose] =
```

Comment:
This has lots of typos

```suggestion
  const [transactionResponse, setTransactionResponse] =
```